### PR TITLE
[khmer_angkor] various updates and fixes

### PR DESCRIPTION
**Pull request by MakaraSok**
* fix the broken link on welcome page
* add example to the keyboard_info
* add tablet layout
* update phone layout
* get rid of the Lunar Dates layer on touch layout
* update documentations
* change OSK font to Khmer Mondulkiri

### CODE REVIEW

**Comment by MakaraSok** [comment]
fix https://github.com/keymanapp/keyboards/issues/1482

**Comment by LornaSIL** [comment]
* The link for the Mondulkiri font should be changed to: https://software.sil.org/mondulkiri/
* Under "More information" in the welcome.htm file there are two things relating to Unicode 10. Since we are at Unicode 13.0 now, it seems appropriate to remove those references, that is:
  * this text `Khmer character chart (Unicode Standard 10.0)` could just be changed to `Khmer character chart`
  * the link for **Khmer Script by Unicode Standard** should be changed from: `http://www.unicode.org/versions/Unicode10.0.0/ch16.pdf` to `http://www.unicode.org/versions/latest/ch16.pdf`
* All the changes to the welcome.htm file (like these links) must also be made in khmer_angkor.php (help file)

**Comment by MakaraSok** [comment]
Thank you for looking into this update.

> * Under "More information" in the welcome.htm file there are two things relating to Unicode 10. Since we are at Unicode 13.0 now, it seems appropriate to remove those references, that is:
>   
>   * this text `Khmer character chart (Unicode Standard 10.0)` could just be changed to `Khmer character chart`
>   * the link for **Khmer Script by Unicode Standard** should be changed from: `http://www.unicode.org/versions/Unicode10.0.0/ch16.pdf` to `http://www.unicode.org/versions/latest/ch16.pdf`
> * All the changes to the welcome.htm file (like these links) must also be made in khmer_angkor.php (help file)

Unicode Standard evolves over time and it might be better to stick to the version considered when the keyboard was created. What do you think?

**Comment by LornaSIL** [comment]
I would link to the latest. The keyboard may not support newer characters,
but the latest gives the most/best description of Khmer.

File: release/k/khmer_angkor/README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Supported Platforms
 -------------------
  * Windows
  * macOS
- *LInux
+ *Linux
```
Comment (mcdurdin) [inline release/k/khmer_angkor/README.md:18]:
```suggestion
 * Linux
```

**Review by mcdurdin — COMMENTED** [review]
I think Lorna captured everything that was important. The changes look good.

**Review by MakaraSok — COMMENTED** [review]
LGTM

**Review by LornaSIL — APPROVED** [review]
LGTM

**Pull request merged**